### ISSUE TITLE
Add accessibilityIdentifier to EKButtonView init

### ISSUE THREAD

**Issue by smileypop** [huri000/SwiftEntryKit#191]
**Is your feature request related to a problem? Please describe.**
I use accessibilityIdentifier for UI tests and enhanced usability for users with disabilities. However, I cannot assign the accessibilityIdentifier to the UIButton that is hidden inside EKButtonView.

**Describe the solution you'd like**
Add `accessibilityIdentifier` as an optional init parameter to ButtonContent.init(), that then gets assigned to the underlying UIButton.

**Describe alternatives you've considered**

**Additional context**

**Comment by huri000** [comment]
@smileypop thanks for noting this subject. `SwiftEntryKit`'s labels, images and buttons are now supporting accessibility (`1.0.4`) 🎉
